kratos/tests/test_processes.py

```diff
@@ -269,87 +319,268 @@ def test_apply_custom_function_process(self):
+
+        for process in list_of_processes:
+            process.ExecuteFinalizeSolutionStep()
+
+        
+    def test_assign_scalar_value_to_conditions(self):
         model_part = ModelPart("Main")
         model_part.AddNodalSolutionStepVariable(NODAL_H)
         model_part_io = ModelPartIO(GetFilePath("test_processes"))
         model_part_io.ReadModelPart(model_part)
+
+        settings = Parameters(
+            """
+            {
+                "process_list" : [
+                    {
+                        "python_module"   : "assign_scalar_to_conditions_process",
+                        "kratos_module" : "KratosMultiphysics",
+                        "process_name"          : "AssignValueProcess",
+                        "Parameters"            : {
+                            "model_part_name":"Main",
+                            "mesh_id": 0,
+                            "variable_name": "PRESSURE",
+                            "value" : 15.0
+                        }
+                    },
+                    {
+                        "python_module"   : "assign_scalar_to_conditions_process",
+                        "kratos_module" : "KratosMultiphysics",
+                        "process_name"          : "AssignValueProcess",
+                        "Parameters"            : {
+                            "model_part_name":"Main",
+                            "mesh_id": 0,
+                            "variable_name": "VISCOSITY",
+                            "value" : 2
+                        }
+                    }
+                    ]
+                }
+            """
+            )
+
+        Model = {"Main":model_part}
+
+        import process_factory
+        list_of_processes = process_factory.KratosProcessFactory(Model).ConstructListOfProcesses( settings["process_list"] )
+
+        for process in list_of_processes:
+            process.ExecuteInitializeSolutionStep()
+            
+        for cond in model_part.Conditions:
+            self.assertEqual(cond.GetValue(PRESSURE), 15.0)
+            self.assertEqual(cond.GetValue(VISCOSITY), 2)
+
         
+    def test_assign_scalar_field_to_conditions(self):
+        model_part = ModelPart("Main")
+        model_part.AddNodalSolutionStepVariable(NODAL_H)
+        model_part_io = ModelPartIO(GetFilePath("test_processes"))
+        model_part_io.ReadModelPart(model_part)
+
+        settings = Parameters(
+            """
+            {
+                "process_list" : [
+                    {
+                        "python_module"   : "assign_scalar_field_to_conditions_process",
+                        "kratos_module" : "KratosMultiphysics",
+                        "process_name"          : "AssignValueProcess",
+                        "Parameters"            : {
+                            "model_part_name":"Main",
+                            "mesh_id": 0,
+                            "variable_name": "INITIAL_STRAIN",

```

Comment:
INITIAL_STRAIN is a variable of type Vector.
we could use any value of type Vectory at this point, however there are not many of those in the Kratos core. Pick one you like more and i'll use that in the test